lib/rust/ensogl/core/src/display/scene/layer.rs

```diff
@@ -656,46 +664,65 @@ impl LayerModel {
         self.sublayers.borrow().all()
     }
 
-    /// Attach a `layer` as a sublayer.
+    /// Iterate over all sublayers, ordered according to the defined depth-order dependencies. The
+    /// layer sublayers list will be borrowed during the iteration.
+    ///
+    /// Please note that this function does not update the depth-ordering of the layers. Updates are
+    /// performed by calling the `update` method on [`Group`], which usually happens once per
```

Comment:
The term "usually happens once per frame" is not precise enough for the reader IMO.

Reply:
This is a copy of comment from a nearby function that follows the same semantics, I wasn't digging into the details that this is talking about. Do you have any better wording in mind that could be used here?

Reply:
I changed that to "happens at least once per frame", as this is more accurate to what is actually happening.